api/v1alpha1/accesslogging_types.go

```diff
@@ -73,14 +77,18 @@ const (
 //
 // +kubebuilder:validation:XValidation:rule="self.type == 'File' ? has(self.file) : !has(self.file)",message="If AccessLogSink type is File, file field needs to be set."
 // +kubebuilder:validation:XValidation:rule="self.type == 'OpenTelemetry' ? has(self.openTelemetry) : !has(self.openTelemetry)",message="If AccessLogSink type is OpenTelemetry, openTelemetry field needs to be set."
+// +kubebuilder:validation:XValidation:rule="self.type == 'GRPC' ? has(self.grpc) : !has(self.grpc)",message="If AccessLogSink type is GRPC, grpc field needs to be set."
 type ProxyAccessLogSink struct {
 	// Type defines the type of accesslog sink.
-	// +kubebuilder:validation:Enum=File;OpenTelemetry
+	// +kubebuilder:validation:Enum=File;GRPC;OpenTelemetry
```

Comment:
grpc is a protocol name, perfer to name it as ALS or something like that.

Reply:
good point, and technically speaking OTel is a gRPC logger just a different API. I like using just `ALS`. I'll make this change!